docs2/site/docs/guides/document-caching.md

```diff
@@ -0,0 +1,66 @@
+# Document Caching
+
+In order to process a GraphQL request, the incoming request must be parsed and validated prior to execution. The parsed and validated
+request may be cached in order to save execution time the next time the same request is executed. For usage patterns where the same
```

Comment:
We need to explain what _the same_ requests mean. With or without operation name, with or without variables.

Reply:
```suggestion
document may be cached in order to save execution time the next time the same document is executed. Note that the
request may select a different operation or have different variables but still be the same document. For usage patterns where the same
```